ao.php

```diff
@@ -146,6 +146,16 @@ function ao_civicrm_validateForm($formName, &$fields, &$files, &$form, &$errors)
   }
 }
 
+function ao_civicrm_check(&$messages) {
+  $current_user = \Drupal::currentUser();
+  if (!empty($current_user)) {
+    $roles = $current_user->getRoles();
+    if (in_array('senior_staff', $roles) || in_array('staff', $roles)) {
```

Comment:
Actually, I think it's best we don't check the role here. Since JMA or a sysadmin is responsible for changes required in CiviCRM, don't show messages for any logged in user.

Reply:
@Edzelopez done .. updated the patch.